server/models/postgis/project.py

```diff
@@ -64,6 +64,10 @@ class Project(db.Model):
     organisation_tag = db.Column(db.String, index=True)
     campaign_tag = db.Column(db.String, index=True)
 
+    # Editors
+    map_editors = db.Column(ARRAY(db.Integer), default=[0, 1, 2, 3], index=True, nullable=False)
+    validate_editors = db.Column(ARRAY(db.Integer), default=[0, 1, 2, 3], index=True, nullable=False)
```

Comment:
same for naming; also, how about using the Editors Enum values below for setting the defaults?

Reply:
Do you mean just replacing the ints with `Editors[editor]` or making it an `ARRAY(Enum(Editors))`?

Reply:
the former, e.g. default=[Editors.ID.value, ...]

Reply:
Fixed 23c4bb4